Android image with espresso automation won't start

When trying to execute tests with espresso automation created image fails to start espresso server.

Output from selenoid/android:10.0 image:
```
2020-08-05 13:27:53:219 - [Gradle] [STDOUT] ............................................................................................................................
2020-08-05 13:27:53:219 - [Gradle] [STDOUT] Unzipping /root/.gradle/wrapper/dists/gradle-5.1.1-all/97z1ksx6lirer3kbvdnh7jtjg/gradle-5.1.1-all.zip to /root/.gradle/wrapper/dists/gradle-5.1.1-all/97z1ksx6lirer3kbvdnh7jtjg
2020-08-05 13:27:54:755 - [Gradle] [STDOUT] Set executable permissions for: /root/.gradle/wrapper/dists/gradle-5.1.1-all/97z1ksx6lirer3kbvdnh7jtjg/gradle-5.1.1/bin/gradle
2020-08-05 13:27:55:029 - [Gradle] [STDOUT] 
2020-08-05 13:27:55:030 - [Gradle] [STDOUT] Welcome to Gradle 5.1.1!
2020-08-05 13:27:55:030 - [Gradle] [STDOUT] 
2020-08-05 13:27:55:030 - [Gradle] [STDOUT] Here are the highlights of this release:
2020-08-05 13:27:55:030 - [Gradle] [STDOUT]  - Control which dependencies can be retrieved from which repositories
2020-08-05 13:27:55:030 - [Gradle] [STDOUT]  - Production-ready configuration avoidance APIs
2020-08-05 13:27:55:030 - [Gradle] [STDOUT] 
2020-08-05 13:27:55:030 - [Gradle] [STDOUT] For more details see https://docs.gradle.org/5.1.1/release-notes.html
2020-08-05 13:27:55:030 - [Gradle] [STDOUT] 
2020-08-05 13:27:55:030 - [Gradle] [STDOUT] Starting a Gradle Daemon (subsequent builds will be faster)
2020-08-05 13:28:15:727 - [Gradle] [STDOUT] 
2020-08-05 13:28:15:728 - [Gradle] [STDOUT] > Configure project :app
2020-08-05 13:28:15:728 - [Gradle] [STDOUT] File /root/.android/repositories.cfg could not be loaded.
2020-08-05 13:28:15:728 - [Gradle] [STDOUT] Checking the license for package Android SDK Build-Tools 28.0.3 in /opt/android-sdk-linux/licenses
2020-08-05 13:28:15:728 - [Gradle] [STDOUT] License for package Android SDK Build-Tools 28.0.3 accepted.
2020-08-05 13:28:15:728 - [Gradle] [STDOUT] Preparing "Install Android SDK Build-Tools 28.0.3 (revision: 28.0.3)".
2020-08-05 13:28:17:227 - [Gradle] [STDOUT] "Install Android SDK Build-Tools 28.0.3 (revision: 28.0.3)" ready.
2020-08-05 13:28:17:227 - [Gradle] [STDOUT] Installing Android SDK Build-Tools 28.0.3 in /opt/android-sdk-linux/build-tools/28.0.3
2020-08-05 13:28:17:227 - [Gradle] [STDOUT] "Install Android SDK Build-Tools 28.0.3 (revision: 28.0.3)" complete.
2020-08-05 13:28:17:227 - [Gradle] [STDOUT] "Install Android SDK Build-Tools 28.0.3 (revision: 28.0.3)" finished.
2020-08-05 13:28:17:227 - [Gradle] [STDOUT] Checking the license for package Android SDK Platform 28 in /opt/android-sdk-linux/licenses
2020-08-05 13:28:17:227 - [Gradle] [STDOUT] License for package Android SDK Platform 28 accepted.
2020-08-05 13:28:17:228 - [Gradle] [STDOUT] Preparing "Install Android SDK Platform 28 (revision: 6)".
2020-08-05 13:28:20:527 - [Gradle] [STDOUT] "Install Android SDK Platform 28 (revision: 6)" ready.
2020-08-05 13:28:20:527 - [Gradle] [STDOUT] Installing Android SDK Platform 28 in /opt/android-sdk-linux/platforms/android-28
2020-08-05 13:28:20:528 - [Gradle] [STDOUT] "Install Android SDK Platform 28 (revision: 6)" complete.
2020-08-05 13:28:20:528 - [Gradle] [STDOUT] "Install Android SDK Platform 28 (revision: 6)" finished.
2020-08-05 13:28:21:727 - [Gradle] [STDOUT] 
2020-08-05 13:28:21:727 - [Gradle] [STDOUT] > Task :app:preBuild UP-TO-DATE
2020-08-05 13:28:21:727 - [Gradle] [STDOUT] > Task :app:preDebugBuild
2020-08-05 13:28:21:727 - [Gradle] [STDOUT] > Task :app:compileDebugAidl NO-SOURCE
2020-08-05 13:28:21:727 - [Gradle] [STDOUT] > Task :app:compileDebugRenderscript NO-SOURCE
2020-08-05 13:28:21:728 - [Gradle] [STDOUT] > Task :app:checkDebugManifest
2020-08-05 13:28:21:728 - [Gradle] [STDOUT] > Task :app:generateDebugBuildConfig
2020-08-05 13:28:21:728 - [Gradle] [STDOUT] > Task :app:mainApkListPersistenceDebug
2020-08-05 13:28:21:728 - [Gradle] [STDOUT] > Task :app:generateDebugResValues
2020-08-05 13:28:21:728 - [Gradle] [STDOUT] > Task :app:generateDebugResources
2020-08-05 13:28:22:227 - [Gradle] [STDOUT] > Task :app:mergeDebugResources
2020-08-05 13:28:22:227 - [Gradle] [STDOUT] > Task :app:createDebugCompatibleScreenManifests
2020-08-05 13:28:22:327 - [Gradle] [STDOUT] > Task :app:processDebugManifest
2020-08-05 13:28:22:427 - [Gradle] [STDOUT] > Task :app:processDebugResources
2020-08-05 13:28:22:923 - [Gradle] [STDOUT] > Task :app:compileDebugKotlin FAILED
2020-08-05 13:28:22:923 - [Gradle] [STDERR] 
2020-08-05 13:28:22:923 - [Gradle] [STDERR] FAILURE: Build failed with an exception.
2020-08-05 13:28:22:923 - [Gradle] [STDERR] 
2020-08-05 13:28:22:923 - [Gradle] [STDERR] * What went wrong:
2020-08-05 13:28:22:923 - [Gradle] [STDERR] Execution failed for task ':app:compileDebugKotlin'.
2020-08-05 13:28:22:923 - [Gradle] [STDERR] > Kotlin could not find the required JDK tools in the Java installation '/usr/lib/jvm/java-8-openjdk-amd64/jre' used by Gradle. Make sure Gradle is running on a JDK, not JRE.
2020-08-05 13:28:22:924 - [Gradle] [STDERR] 
2020-08-05 13:28:22:924 - [Gradle] [STDERR] * Try:
2020-08-05 13:28:22:924 - [Gradle] [STDERR] Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
2020-08-05 13:28:22:924 - [Gradle] [STDERR] 
2020-08-05 13:28:22:924 - [Gradle] [STDOUT] 
2020-08-05 13:28:22:924 - [Gradle] [STDOUT] Deprecated Gradle features were used in this build, making it incompatible with Gradle 6.0.
2020-08-05 13:28:22:924 - [Gradle] [STDOUT] Use '--warning-mode all' to show the individual deprecation warnings.
2020-08-05 13:28:22:924 - [Gradle] [STDOUT] See https://docs.gradle.org/5.1.1/userguide/command_line_interface.html#sec:command_line_warnings
2020-08-05 13:28:22:924 - [Gradle] [STDOUT] 10 actionable tasks: 10 executed
2020-08-05 13:28:22:924 - [Gradle] [STDERR] * Get more help at https://help.gradle.org
2020-08-05 13:28:22:924 - [Gradle] [STDERR] 
2020-08-05 13:28:22:924 - [Gradle] [STDERR] BUILD FAILED in 32s
2020-08-05 13:28:23:263 - [Espresso] Unable to build Espresso server - Process ended with exitcode 1 (cmd: './gradlew assembleAndroidTest')
2020-08-05 13:28:23:264 - [Espresso] Gradle error message:
2020-08-05 13:28:23:264 - [Espresso] 
2020-08-05 13:28:23:264 - [Espresso] [STDERR] * What went wrong:,
2020-08-05 13:28:23:264 - [Espresso] [STDERR] Execution failed for task ':app:compileDebugKotlin'.,
2020-08-05 13:28:23:264 - [Espresso] [STDERR] > Kotlin could not find the required JDK tools in the Java installation '/usr/lib/jvm/java-8-openjdk-amd64/jre' used by Gradle. Make sure Gradle is running on a JDK, not JRE.,
2020-08-05 13:28:23:264 - [Espresso] [STDERR] ,
2020-08-05 13:28:23:264 - [Espresso] [STDERR] * Try:,
2020-08-05 13:28:23:264 - [Espresso] [STDERR] Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.,
2020-08-05 13:28:23:264 - [Espresso] [STDERR] ,
2020-08-05 13:28:23:264 - [Espresso] [STDOUT] ,
2020-08-05 13:28:23:264 - [Espresso] [STDOUT] Deprecated Gradle features were used in this build, making it incompatible with Gradle 6.0.,
2020-08-05 13:28:23:264 - [Espresso] [STDOUT] Use '--warning-mode all' to show the individual deprecation warnings.,
2020-08-05 13:28:23:264 - [Espresso] [STDOUT] See https://docs.gradle.org/5.1.1/userguide/command_line_interface.html#sec:command_line_warnings,
2020-08-05 13:28:23:264 - [Espresso] [STDOUT] 10 actionable tasks: 10 executed,
2020-08-05 13:28:23:264 - [Espresso] [STDERR] * Get more help at https://help.gradle.org,
2020-08-05 13:28:23:264 - [Espresso] [STDERR] ,
2020-08-05 13:28:23:265 - [Espresso] [STDERR] BUILD FAILED in 32s
2020-08-05 13:28:23:265 - [debug] [Espresso] Deleting espresso session
2020-08-05 13:28:23:265 - [debug] [ADB] Running '/opt/android-sdk-linux/platform-tools/adb -P 5037 -s emulator-5554 shell am broadcast -a io.appium.settings.animation -n io.appium.settings/.receivers.AnimationSettingReceiver --es setstatus enable'
2020-08-05 13:28:23:308 - [debug] [ADB] Running '/opt/android-sdk-linux/platform-tools/adb -P 5037 -s emulator-5554 shell am force-stop org.worldbank.solutions.interviewer'
2020-08-05 13:28:23:331 - [debug] [Logcat] Stopping logcat capture
2020-08-05 13:28:23:331 - [Espresso] Restoring hidden api policy to the device default configuration
2020-08-05 13:28:23:331 - [debug] [ADB] Running '/opt/android-sdk-linux/platform-tools/adb -P 5037 -s emulator-5554 shell settings delete global hidden_api_policy_pre_p_apps'
2020-08-05 13:28:23:352 - [debug] [ADB] Running '/opt/android-sdk-linux/platform-tools/adb -P 5037 -s emulator-5554 shell settings delete global hidden_api_policy_p_apps'
2020-08-05 13:28:23:369 - [debug] [ADB] Running '/opt/android-sdk-linux/platform-tools/adb -P 5037 -s emulator-5554 shell settings delete global hidden_api_policy'
2020-08-05 13:28:23:387 - [debug] [ADB] Removing forwarded port socket connection: 8300 
2020-08-05 13:28:23:388 - [debug] [ADB] Running '/opt/android-sdk-linux/platform-tools/adb -P 5037 -s emulator-5554 forward --remove tcp\:8300'
2020-08-05 13:28:23:394 - [debug] [BaseDriver] Event 'newSessionStarted' logged at 1596634103394 (13:28:23 GMT+0000 (Coordinated Universal Time))
2020-08-05 13:28:23:394 - [debug] [W3C] Encountered internal error running command: A new session could not be created. Details: Unable to build Espresso server - Process ended with exitcode 1 (cmd: './gradlew assembleAndroidTest')
2020-08-05 13:28:23:394 - [debug] [W3C] Gradle error message:
2020-08-05 13:28:23:394 - [debug] [W3C] 
2020-08-05 13:28:23:394 - [debug] [W3C] [STDERR] * What went wrong:,
2020-08-05 13:28:23:394 - [debug] [W3C] [STDERR] Execution failed for task ':app:compileDebugKotlin'.,
2020-08-05 13:28:23:394 - [debug] [W3C] [STDERR] > Kotlin could not find the required JDK tools in the Java installation '/usr/lib/jvm/java-8-openjdk-amd64/jre' used by Gradle. Make sure Gradle is running on a JDK, not JRE.,
2020-08-05 13:28:23:394 - [debug] [W3C] [STDERR] ,
2020-08-05 13:28:23:394 - [debug] [W3C] [STDERR] * Try:,
2020-08-05 13:28:23:394 - [debug] [W3C] [STDERR] Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.,
2020-08-05 13:28:23:394 - [debug] [W3C] [STDERR] ,
2020-08-05 13:28:23:395 - [debug] [W3C] [STDOUT] ,
2020-08-05 13:28:23:395 - [debug] [W3C] [STDOUT] Deprecated Gradle features were used in this build, making it incompatible with Gradle 6.0.,
2020-08-05 13:28:23:395 - [debug] [W3C] [STDOUT] Use '--warning-mode all' to show the individual deprecation warnings.,
2020-08-05 13:28:23:395 - [debug] [W3C] [STDOUT] See https://docs.gradle.org/5.1.1/userguide/command_line_interface.html#sec:command_line_warnings,
2020-08-05 13:28:23:395 - [debug] [W3C] [STDOUT] 10 actionable tasks: 10 executed,
2020-08-05 13:28:23:395 - [debug] [W3C] [STDERR] * Get more help at https://help.gradle.org,
2020-08-05 13:28:23:395 - [debug] [W3C] [STDERR] ,
2020-08-05 13:28:23:395 - [debug] [W3C] [STDERR] BUILD FAILED in 32s. Check https://github.com/appium/appium-espresso-driver#troubleshooting regarding advanced session startup troubleshooting.
2020-08-05 13:28:23:395 - [debug] [W3C] Error: Unable to build Espresso server - Process ended with exitcode 1 (cmd: './gradlew assembleAndroidTest')
2020-08-05 13:28:23:395 - [debug] [W3C] Gradle error message:
2020-08-05 13:28:23:395 - [debug] [W3C] 
2020-08-05 13:28:23:395 - [debug] [W3C] [STDERR] * What went wrong:,
2020-08-05 13:28:23:395 - [debug] [W3C] [STDERR] Execution failed for task ':app:compileDebugKotlin'.,
2020-08-05 13:28:23:395 - [debug] [W3C] [STDERR] > Kotlin could not find the required JDK tools in the Java installation '/usr/lib/jvm/java-8-openjdk-amd64/jre' used by Gradle. Make sure Gradle is running on a JDK, not JRE.,
2020-08-05 13:28:23:395 - [debug] [W3C] [STDERR] ,
2020-08-05 13:28:23:395 - [debug] [W3C] [STDERR] * Try:,
2020-08-05 13:28:23:395 - [debug] [W3C] [STDERR] Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.,
2020-08-05 13:28:23:395 - [debug] [W3C] [STDERR] ,
2020-08-05 13:28:23:396 - [debug] [W3C] [STDOUT] ,
2020-08-05 13:28:23:396 - [debug] [W3C] [STDOUT] Deprecated Gradle features were used in this build, making it incompatible with Gradle 6.0.,
2020-08-05 13:28:23:396 - [debug] [W3C] [STDOUT] Use '--warning-mode all' to show the individual deprecation warnings.,
2020-08-05 13:28:23:396 - [debug] [W3C] [STDOUT] See https://docs.gradle.org/5.1.1/userguide/command_line_interface.html#sec:command_line_warnings,
2020-08-05 13:28:23:396 - [debug] [W3C] [STDOUT] 10 actionable tasks: 10 executed,
2020-08-05 13:28:23:396 - [debug] [W3C] [STDERR] * Get more help at https://help.gradle.org,
2020-08-05 13:28:23:396 - [debug] [W3C] [STDERR] ,
2020-08-05 13:28:23:396 - [debug] [W3C] [STDERR] BUILD FAILED in 32s. Check https://github.com/appium/appium-espresso-driver#troubleshooting regarding advanced session startup troubleshooting.
2020-08-05 13:28:23:397 - [debug] [W3C]     at Object.errorAndThrow (/opt/node_modules/appium/node_modules/appium-support/lib/logging.js:74:35)
2020-08-05 13:28:23:397 - [debug] [W3C]     at ServerBuilder.runBuildProcess (/opt/node_modules/appium/node_modules/appium-espresso-driver/lib/server-builder.js:152:11)
2020-08-05 13:28:23:398 - [HTTP] <-- POST /wd/hub/session 500 41276 ms - 2713

```

--- a/selenium/android/Dockerfile
+++ b/selenium/android/Dockerfile
@@ -22,7 +22,7 @@ RUN \
 	    curl \
 	    iproute2 \
 	    nodejs \
-	    openjdk-8-jre-headless \
+	    openjdk-8-jdk-headless \
 	    unzip \
 	    xvfb \
 	    libpulse0 \

